extension/communicate-to-arcs.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+function _getUrl(cdnRoot, arcManifest, amKey) {
+  return cdnRoot+'?manifest='+encodeURIComponent(arcManifest)+'&amkey='+amKey;
+}
+
+function populateIframe(doc) {
+
+  var iframe = doc.getElementById('arcs-if');
+  let cdnRoot = 'https://polymerlabs.github.io/arcs-cdn/dev/app/';
```

Comment:
Do we have a plan to not hard code all of these urls? For debugging this it would be handy to run against a local instance of the CDN.

Reply:
Unfortunately I don't have a great plan. I've moved them out into a globals.js (and I'll include sample values, commented out, for using a local CDN) so at least it's easy to see where to make changes.

It would be nice to do better still, but nothing has occurred to me. Any thoughts?